grunt-tasks/purifycss.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("purifycss", {
+    options: {
+      rejected: true,
```

Comment:
So the idea is we create a stylesheet that *only* contains the unused styles? And then presumably we send in PRs from time to time removing them?

Could we add something somewhere in docs to explain this?

Reply:
(if we're crunched on time, filing a follow up bug to document is this is also cool)

Reply:
No, we create one without the removed styles. The `rejected` option is showing in the logs the unsused styles that have been removed.

Reply:
Ah, thanks!